tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -425,6 +425,17 @@ ts_decompress_chunk_generate_paths(PlannerInfo *root, RelOptInfo *chunk_rel, Hyp
 		Path *child_path = lfirst(lc);
 		DecompressChunkPath *path;
 
+		/*
+		 * We skip any BitmapScan paths here as supporting those
+		 * would require fixing up the internal scan. Since we
+		 * currently do not do this BitmapScans would be generated
+		 * when we have a parameterized path on a compressed column
+		 * that would have invalid references due to our
```

Comment:
Does bitmap scan work for the non-parameterized path? If so, maybe we should move this condition under the `if (child_path->param_info != NULL)`.